automation/taskcluster/decision_task_nightly.py

```diff
@@ -53,7 +53,7 @@ def generate_build_task(apks, is_staging):
             checkout +
             ' && python automation/taskcluster/helper/get-secret.py'
             ' -s {} -k dsn -f .sentry_token'.format(sentry_secret) +
-            ' && ./gradlew --no-daemon -PcrashReports=true clean test assembleGreenfieldRelease'),
```

Comment:
Out of curiosity, what's the error like when using that flavor?

Reply:
> FAILURE: Build failed with an exception.
> 
> * What went wrong:
> Task 'assembleGreenfieldRelease' not found in root project 'fenix'. Some candidates are: 'assembleArmGreenfieldRelease', 'assembleX86GreenfieldRelease'.